wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -2134,6 +2158,15 @@ image will be updated automatically and independent from calls to GetSnapshotUri
 				<soap:body parts="parameters" use="literal"/>
 			</wsdl:output>
 		</wsdl:operation>
+		<wsdl:operation name="SetEQPreset">
```

Comment:
should we have a GetEQPreset? Even thoug it is a capability what is possible, if setting a EQ for the editable ones, I guess that won't show up in capabilities and instead need to be retrieved via a get-request?

Reply:
> should we have a GetEQPreset? Even thoug it is a capability what is possible, if setting a EQ for the editable ones, I guess that won't show up in capabilities and instead need to be retrieved via a get-request?

@robberos, Initially, we planned to include geteqpreset. However, during our previous discussions, some members preferred not to add more methods. Instead, we now provide the list of eqpreset in the GetAudioOutputConfigurationOptions response. If you believe this method is necessary, we can discuss it in the next telco.